AssignmentTests/SampleDataTests.cs

```diff
@@ -0,0 +1,92 @@
+using Assignment;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+
+namespace AssignmentTests
+{
+    [TestClass]
+    public class SampleDataTests
+    {
+        [TestMethod]
+        public void SampleData_CsvRows_ReturnsListFirstLineSkipped()
+        {
+            // Arrange
+            SampleData data = new SampleData();
+            int lineCount = File.ReadAllLines("People.csv").Count();
+
+            // Act
+
+
+            // Assert
+            Assert.AreEqual(data.CsvRows.Count(), lineCount - 1);
+        }
+
+        [TestMethod]
+        public void SampleData_GetUniqueSortedListOfStatesGivenCsvRows_ReturnsSortedUniqueStates()
+        {
+            // Arrange
+            SampleData data = new SampleData();
```

Comment:
Since this is a test method you might want to use your TestPeople.csv. You could do this with your SampleData constructor that takes a "fileName"
```suggestion
            SampleData data = new SampleData(Environment.CurrentDirectory + "//TestPeople.csv");
```